docs/deployments/configuration.md

```diff
@@ -845,3 +920,12 @@ Using the API:
   }]
 }
 ```
+
```

Comment:
@kriszyp I didn't know what to write to describe serialization.bigInt

Reply:
I don't think serialization.bigInt was ever actually implemented, so can you remove this one too.